### DISCUSSION TITLE
Devices have association, but I have never created and can not un-associate

### DISCUSSION THREAD

**Discussion by jamesarbrown** [zwave-js/zwave-js#3685] [Q&A]
Hi,

I am struggling with 2 x MCO Home MH-P220 Dimmers in that one dimmer has a pushbutton on S2 and its triggering a scene on another MH-p220 dimmer

On the dimmer it only has this association
![image](https://user-images.githubusercontent.com/891438/140814580-2c4526f2-7137-425a-b482-58aba68d06e9.png)

With these settings
![image](https://user-images.githubusercontent.com/891438/140814663-91efd0d2-dde9-472d-a700-a2bf244e93b4.png)

But no matter what I do, triggering S2 input on that dimmer triggers the scene on the other dimmer. Even if the other dimmer is OFF, it comes on an sets the scene. Home assistant was stopped to triple check it was not involved.

Have also tried Advanced > Associations > Clear also on both devices through zwavejs2mqtt interface.

Any ideas or debugging I can do and what to look at?

Thanks

**Comment by AlCalzone** [comment]
You can try re-interviewing the device to see if that reveals another association. 
Ideally make [driver log](https://zwave-js.github.io/zwavejs2mqtt/#/troubleshooting/generating-logs?id=driver-logs) of that process, loglevel `debug` or `silly` and attach it here as a file.

**Reply by jamesarbrown** [reply]
Node 25 is the one with the scene trigger. Interestingly if this node is OFF it stays OFF
Node 36 is the one reacting to it, but with no association. Even if it is OFF it comes ON

Node 36 is recently installed and new. No associations have been created.
Node 36+25 are on same firmware

Both nodes I have reinterviewed = no associations other than lifeline as above
Both nodes I have tried "clear associations"

**Driver Log**
```
2021-11-14T18:16:53.924Z CNTRLR   [Node 025] [!] [Scene Activation] sceneId: 1                      [Endpoint 0]
2021-11-14T18:16:53.926Z CNTRLR   [Node 025] [~] [Scene Activation] dimmingDuration: "default" => " [Endpoint 0]
                                  default"
2021-11-14T18:16:53.928Z SERIAL » [ACK]                                                                   (0x06)
2021-11-14T18:16:53.934Z DRIVER « [Node 025] [REQ] [ApplicationCommand]
                                  │ type: broadcast
                                  └─[SceneActivationCCSet]
                                      scene id:         1
                                      dimming duration: default
2021-11-14T18:16:54.260Z SERIAL » 0x01050051240d82                                                     (7 bytes)
2021-11-14T18:16:54.261Z DRIVER » [Node 036] [REQ] [AssignSUCReturnRoute]
                                    payload: 0x240d
2021-11-14T18:16:54.262Z SERIAL « [ACK]                                                                   (0x06)
2021-11-14T18:16:55.867Z CNTRLR   [Node 036] Assigning SUC return route failed: Timeout while waiting for a resp
                                  onse from the controller (ZW0200)
2021-11-14T18:16:56.610Z SERIAL « 0x0104015101aa                                                       (6 bytes)
2021-11-14T18:16:56.611Z SERIAL » [ACK]                                                                   (0x06)
2021-11-14T18:16:56.612Z DRIVER « [RES] [AssignSUCReturnRoute]
                                    was executed: true
2021-11-14T18:16:56.615Z DRIVER « [RES] [AssignSUCReturnRoute]                                      [unexpected]
                                    was executed: true
2021-11-14T18:16:56.616Z DRIVER   unexpected response, discarding...
2021-11-14T18:16:56.625Z SERIAL « 0x010500510d04a2                                                     (7 bytes)
2021-11-14T18:16:56.625Z SERIAL » [ACK]                                                                   (0x06)
2021-11-14T18:16:56.627Z DRIVER « [REQ] [AssignSUCReturnRoute]
                                    callback id:     13
                                    transmit status: NoRoute
2021-11-14T18:16:56.632Z DRIVER   handling request AssignSUCReturnRoute (81)
2021-11-14T18:16:56.632Z DRIVER     no handlers registered!
2021-11-14T18:17:01.619Z SERIAL « 0x01100004001e0a600d07003105012200ea68                              (18 bytes)
2021-11-14T18:17:01.624Z SERIAL » [ACK]    
```

**Reply by AlCalzone** [reply]
I need the entire log, not just a short snippet. Please attach it as a file (drag&drop).

**Reply by jamesarbrown** [reply]
[zwavejs_2021-11-14.log](https://github.com/zwave-js/node-zwave-js/files/7534428/zwavejs_2021-11-14.log)

Note the log was limited to nodes 25+36. I can rerun with all if you need

**Reply by jamesarbrown** [reply]
A bit more information

I have tried, options 1+2 with same results
![image](https://user-images.githubusercontent.com/891438/141743478-674eb5b8-1f27-45c0-a46e-6e54c63e0f77.png)

Also note, I am not even looking for scene. Just looking for a switch in HA that I can use for automations, but basicCCSET does not pass through zwavejs2MQTT, and I dont know how to tweak to do that.

**Reply by AlCalzone** [reply]
> I can rerun with all if you need

Not necessary to rerun with all nodes, but you should at least re-interview these two nodes **while** making the logfile.

> I have tried, options 1+2 with same results

Configuration parameters have nothing to do with associations

**Comment by jamesarbrown** [comment]
[Node36 Reinterview.log](https://github.com/zwave-js/node-zwave-js/files/7541731/Node36.Reinterview.log)
[Node25 Reinterview.log](https://github.com/zwave-js/node-zwave-js/files/7541733/Node25.Reinterview.log)

**Reply by AlCalzone** [reply]
Ok I don't see any "hidden" associations in there, so we can bury that theory. This seems to be caused by broadcasts from one dimmer. I think this is because the dimmer has no route back to the controller and tries reaching it using broadcasts as a hail mary. The attempts to assign a route also failed because none could be determined:
```
2021-11-15T21:20:35.525Z DRIVER « [REQ] [AssignSUCReturnRoute]
                                    callback id:     13
                                    transmit status: NoRoute
```

---

I'd start by making sure the controller doesn't have interference problems. Z-Wave sticks in particular are prone to interference by USB ports, especially by USB3 ports. We recommend putting the stick in a suitable location:

-   on an **USB extension cord** (this works wonders!)
-   away from other USB ports
-   away from metallic surfaces
-   and especially not in the back of a server rack

---

After that you should perform a network heal. This will calculate new routes and might solve your problem already.

**Reply by jamesarbrown** [reply]
Yes..... maybe something in that. There are problems in the mesh, but for the life of me I can not understand why as there are devices in line of sight that should provide a route.

The controller has always been on a USB cable and hung in a better place, but the top floor it is on, the entire roof area is foil coated.

But, the light (node 25) on the outside of the house heals fine tonight (about 12m away from controlle by line of sight)

The node 36, some 17m away from node 25, obstructed only by 18mm of wood wont heal and nor will any nodes in that area. 

And worse still now since I have been trying node 36 wont work, yet another node <1m away is fine.

I think I need to exclude and reinclude some nodes from scratch. If it is interference then its awfully disappointing there should be enough mesh to work around on shorter hops.

**Is there anyway to release older node IDs for re-use?**

**Reply by AlCalzone** [reply]
> Is there anyway to release older node IDs for re-use?

No. Only if you wipe the controller or exclude/include often enough to wrap around after 232.

**Reply by jamesarbrown** [reply]
So a full reset of the stick, a full reset of all devices, upladders, light switches de mounted, new inclusion to them all.... and same *darnit*

This i am beginning to wonder is a bug in the device, after all one device is sending "scene 1" to group association 1 (lifeline) and the other device has no setting (eg off) for "scene 1" and its just picking up on group assoc 1 even though I have not associated it.

That I am sure its happening, if and why it should be doing that.....

Next task is to demount and bring near the controller for a bench test.

**Reply by githubbuh** [reply]
I just installed the current home assistant version and elected to not migrate my 34 z-wave devices. 

I elected to go the scorched-earth route with my z-stick. I removed all of the devices from the z-stick, factory reset all the devices, then went through the process of adding them all back using Z-Wave JS to MQTT. I have 34 devices. It took about 6 hours. While Z-Wave JS to MQTT is a big improvement, it is not without its defects so be prepared to be patient. [zwave js crew, I'll post some issues for you.]

I also spent a couple of hours getting this to work: see the "Remove a Failed Node using Zensys Tool / PC Controller 4 : Aeotec Help Desk" page. The utility that aeotec offers is down a major version, has an unsigned driver for pre-win10, which requires some work to install on pre-win 10 machines. The driver for the z-stick is native in Win10. on Win10 it is necessary to install .net 2 and 2.5 as i recall.  It is very useful for managing dead nodes. I imagine it will work for other vendor products since it is not a utility created by aeotec.

For what it is worth.

**Reply by nesheimt** [reply]
Hi @jamesarbrown 
Did you ever get to the bottom of this? 

I migrated to zwave js ui(on Homeseer4) some time ago and associations seems not to function very smoothly. Lots of associations "appared" without my involvement, and I cannot add/delete anything on some devices. "Clear" og "remove all" don't work either. This issue seems to be connected to come devices especially Heatit thermostats and Shelly actuators.
